scripts/update/Updater.php

```diff
@@ -428,5 +428,7 @@ public function update($currentVersion)
         }
 
         $this->skip('6.6.0', '6.6.1');
+
+        $this->skip('6.6.1', '6.7.1');
```

Comment:
A new `$this->skip()` call is not necessary. You can change the existing `skip('6.6.0', '6.6.1')` to `skip('6.6.0', '6.7.0')`.

Reply:
Nice trick, thanks @bugalot !